www/src/data/sidebars/doc-links.yaml

```diff
@@ -219,6 +219,8 @@
               link: /docs/adding-search-with-elasticlunr/
             - title: Adding Search with js-search
               link: /docs/adding-search-with-js-search/
+        - title: Adding Comments
```

Comment:
Want to move this down a bit? e.g. below "Adding Pagination" perhaps?

Reply:
Sure thing.